### ISSUE TITLE
When trying to open large blueprints I get this error.

### ISSUE THREAD

**Issue by Inspiredememy** [gabriel-dehan/dyson-sphere-blueprints#32]
When trying to open large blueprints I get this error.
![image](https://user-images.githubusercontent.com/91331968/134689676-2d619211-b071-4ad3-b269-b47d06bc2c12.png)

**Comment by gabriel-dehan** [comment]
Hey man, could you link me the blueprint in question? Thanks

**Comment by Inspiredememy** [comment]
This is the one I was trying.
https://www.dysonsphereblueprints.com/blueprints/science-planet-2500-min-from-raw

Seems to happen when its loading the text for the blueprint field.

**Comment by gabriel-dehan** [comment]
@Inspiredememy yeah that's just a way too big blueprint (100k structures) and loading simply takes too long. You can choose to wait and eventually it will load though, just tried it. It took a good few minutes.
I am not sure yet what's the best way of going about those very big blueprint that take forever to load, I am considering a few options atm :)
